lib/core/test/unit/Cardano/Wallet/Primitive/TypesSpec.hs

```diff
@@ -302,6 +323,63 @@ prop_2_6_2 (ins, u) =
             ===
         balance u - balance (u `restrictedBy` ins)
 
+{-------------------------------------------------------------------------------
+                        UTxO statistics Properties
+-------------------------------------------------------------------------------}
+
+-- | The 'total' stake in the statistics is the UTxO's balance
+propUtxoTotalIsBalance
+    :: BoundType
+    -> ShowFmt UTxO
+    -> Property
+propUtxoTotalIsBalance bType (ShowFmt utxo) =
+    fromIntegral totalStake == balance utxo
+    & cover 75 (utxo /= mempty) "UTxO /= empty"
+  where
+    UTxOStatistics _ totalStake _ = computeUtxoStatistics bType utxo
+
+-- | The sum of the weighted distribution is greater than the sum of UTxOs
```

Comment:
> weighted distribution is greater than the sum of UTxOs

I don't get this at all… Why is this a **weighted** distribution? If it is, it's not relevant here?

But ok, the bucket "10" really contains "1-10", so multiplying buckets values with counts should `>=`.

Reply:
Each value is in a "bucket". So, if we computed a weighted sum where the weights are the bucket upper-bound, we get a value that should be bigger than the original value distributed over all buckets. That's what called a "weighted distribution" in short :sweat_smile: